packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -46,13 +46,23 @@ interface IMultiselectProps extends HTMLAttributes<HTMLDivElement> {
   validation?: VALIDATION;
   /** Determines the maximum number of items displayed while collapsed */
   maxItems?: number;
-  /** Callback that overrides the "+ N more" text displayed when the total number of items exceeds `maxItems` */
+  /**
+   * Overrides the "+ N more" text displayed when the total number of items exceeds `maxItems`
+   *
+   * @param {number} index
+   * @returns {string} A replacement for the "+ N more" text
+   */
   renderShowMore?: (index: number) => string;
-  /** Callback that renders each item element */
+  /**
+   * Renders each item element
```

Comment:
We could consider:

`Renders each item element. Designed to be used with [Tag](https://garden.zendesk.com/components/tags).`